### DISCUSSION TITLE
WhiteField is not getting configured on Ubuntu 20.04.

### DISCUSSION THREAD

**Discussion by nkellie813** [whitefield-framework/whitefield#129] [General]
Error File is attached herewith
[WhiteFiled Error 20.04LTS.txt](https://github.com/whitefield-framework/whitefield/files/7231069/WhiteFiled.Error.20.04LTS.txt)

**Comment by nyrahul** [comment]
I think there was a problem in setup.sh that might have caused compilation problem. Have fixed it in #131 . Also added vagrant setup instruction in the `README.md`.

This should now work for your case.
